src/utils/cql/evaluateCql.js

```diff
@@ -0,0 +1,62 @@
+const { Client } = require('cql-translation-service-client');
+const { Library, Executor, Repository } = require('cql-execution');
+const { PatientSource } = require('cql-exec-fhir');
+const atob = require('atob');
+const axios = require('axios');
+const { ValueSetLoader } = require('./ValueSetLoader');
+const configUtil = require('../../storage/configUtil');
+
+async function evaluateCQL(resources, expression, library, patientId) {
+  const elm = await fetchELM(library);
```

Comment:
So this works but just as a thought we could convert the CQL to ELM when the library is added to the database. And then do a check here to see if the elm exists

Reply:
Added this.

Reply:
One additional pieces - if `library.elm` is not set here then when it is converted it could be updated in the database. We expect it to always have the elm so perhaps this is overkill (for instance the elm isn't set yet because it's still being converted so there's no point trying again)

Reply:
Possibly overkill but I added the database upsert to the `fetchElm` function.